backend/src/main/kotlin/blue/mild/covid/vaxx/monitoring/JsonLoggingLayout.kt

```diff
@@ -24,7 +24,7 @@ class JsonLoggingLayout : LayoutBase<ILoggingEvent>() {
 
     override fun doLayout(event: ILoggingEvent): String {
         val finalMap: MutableMap<String, Any> = mutableMapOf(
-            "@timestamp" to formatTime(event),
+            "timestamp" to formatTime(event),
```

Comment:
to jsem myslel, ze mas nachval `@`

Reply:
no to tam bylo kvuli elasticu, ale amazonu se to nelibi a ukazuje tam `@@timestamp`